test/acceptance/behave/components/ngsiv2/base/environment.py

```diff
@@ -0,0 +1,96 @@
+# -*- coding: utf-8 -*-
+"""
+ Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
+
+ This file is part of Orion Context Broker.
+
+ Orion Context Broker is free software: you can redistribute it and/or
+ modify it under the terms of the GNU Affero General Public License as
+ published by the Free Software Foundation, either version 3 of the
+ License, or (at your option) any later version.
+
+ Orion Context Broker is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+ General Public License for more details.
+
+ You should have received a copy of the GNU Affero General Public License
+ along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+
+ For those usages not covered by this license please contact with
+ iot_support at tid dot es
+"""
+__author__ = 'Iván Arias León (ivan dot ariasleon at telefonica dot com)'
+
+
+from components.common_steps.initial_steps import *
+
+FORMAT = "%(asctime)s %(name)s %(message)s"
+logging.basicConfig(format=FORMAT,filename="logs/qa.log", level=logging.DEBUG)
```

Comment:
You can setup the logging format and level in your behave.ini file with the properties **--logging-format** and **--logging-level**:

       --logging-format: Specify custom format to print statements. Uses the same format as used 
                         by standard logging handlers. The default is ‘%(levelname)s:%(name)s:%(message)s’.
       --logging-level:  Specify a level to capture logging at. The default is INFO - capturing everything.

I think you could configure all logging properties (output file, etc) using a logging.ini file. Take a look to this example: _https://github.com/behave/behave/blob/master/features/logging.setup_with_configfile.feature_